src/Common/src/Interop/Windows/mincore/Interop.CreateFile.cs

```diff
@@ -18,5 +19,18 @@ internal partial class mincore
             System.IO.FileMode dwCreationDisposition,
             int dwFlagsAndAttributes,
             IntPtr hTemplateFile);
+
+        internal static SafeFileHandle CreateFile(
```

Comment:
For my information what happens if you get a FileHandle to a long path file? If you query the full path for it will it have the extended path prefix?